tests/api/test_datapipelines.py

```diff
@@ -22,6 +25,9 @@ def env2(env, org1, user, group, tenant, module_mocker):
     module_mocker.patch(
         'dataall.api.Objects.Environment.resolvers.check_environment', return_value=True
     )
+    module_mocker.patch(
+        'dataall.api.Objects.Environment.resolvers.get_pivot_role_as_part_of_environment', return_value=False
```

Comment:
I see we repeat it in so many places, maybe we can mock just somewhere in one place like a fixture with autoUse set to true?